dspace-rest/src/main/java/org/dspace/rest/filter/ItemFilterDefs.java

```diff
@@ -99,12 +99,12 @@ public boolean testItem(Context context, Item item) {
 	    },
 	    has_pdf_original("Item has a PDF Original Bitstream", null, CAT_MIME) {
 	        public boolean testItem(Context context, Item item) {
-	        	return ItemFilterUtil.countOriginalBitstreamMime(context, item, "application/pdf") > 0;
+	        	return ItemFilterUtil.countOriginalBitstreamMime(context, item, "application/pdf".split(",")) > 0;
```

Comment:
Looks good overall, but to make it clearer here, it might be worth chaning these `split()` calls to literal arrays: `{"application/pdf"}`